### PR TITLE
feat: hidden value plugins

### PR DESCRIPTION
**Pull request by 1pone**
Export `HiddenValue` value plugins, to fix the inability to customize value components for version vanilla

### CODE REVIEW

File: src/lib/plugins/value/components/HiddenValue.svelte [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      // open in edit mode
+      onSelect(createValueSelection(path, true))
+    }
+  }
+</script>
+
+<div
+  role="button"
+  tabindex="-1"
+  class="jse-value jse-readonly-password"
```
Comment (josdejong) [inline src/lib/plugins/value/components/HiddenValue.svelte:29]: Can you rename the class to `jse-hidden-value`?

File: src/lib/index.ts [hunk 2]
```diff
@@ -13,7 +14,15 @@ export * from './types.js'
 // value plugins
 export { renderValue } from './plugins/value/renderValue.js'
 export { renderJSONSchemaEnum } from './plugins/value/renderJSONSchemaEnum.js'
-export { BooleanToggle, ColorPicker, EditableValue, EnumValue, ReadonlyValue, TimestampTag }
+export {
```
Comment (josdejong) [inline src/lib/index.ts:17]: Can you also mention this new component in the README?

https://github.com/josdejong/svelte-jsoneditor/blob/main/README.md?plain=1#L286

**Review by josdejong — COMMENTED** [review]
Thanks Jesse! 

I made two small inline comments, can you have a look at those?

**Comment by josdejong** [comment]
I had a look into the long term solution that is discussed in #375 to make it possible to create components in vanilla JS, and it turns out using Svelte actions works like a charm, see #398.

If we have that solution working, this `HiddenValue` component is probably not needed: you can implement it yourself then (and refine the experience to make it a true password component). What do you think?

**Comment by 1pone** [comment]
Wonderful! I'll try to implement it by myself in vanilla JS as soon as it's officially released. At that point, I'll close this pr if it works.

**Comment by josdejong** [comment]
I've finished and merged #398 (not yet published)

**Pull request closed without merging**